src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -198,15 +195,34 @@ private void AddPackageItemsToDictionary(ref Dictionary<string, PackageItem> pac
             }
         }
 
-        // A versions file is of the form https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Latest_Packages.txt
+        private static async System.Threading.Tasks.Task<string> GetFileAsync(string uri)
+        {
+            string contents = string.Empty;
+            using (var stream = (File.Exists(uri)) ? File.OpenRead(uri) : await new HttpClient().GetStreamAsync(uri))
+            using (StreamReader reader = new StreamReader(stream))
+            {
+                contents = await reader.ReadToEndAsync();
+            }
+            return contents;
+        }
+
+        // A versions file is of the form https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/master/Latest_Packages.txt
```

Comment:
It's manual at the moment, something to do on release. The idea is that we have [build pointers](https://github.com/dotnet/versions#build-pointers) for build qualities, but we haven't automated it yet. (LKG would probably be for a green test build.)

A while back there was some discussion about if it makes more sense for `Latest_Packages` to be a list where packages aren't "removed" when the next build publishes, and have a separate list for only the last build's packages. That hasn't happened--but is that kind of Latest what would be useful here?